pennylane/__init__.py

```diff
@@ -78,6 +78,9 @@
     simplify,
 )
 from pennylane.ops.functions import *
+from pennylane.ops import convert
+from pennylane.ops.convert import import_operator
```

Comment:
The module `convert.py` which hosts functions for converting openfermion operator objects to pennylane objects is moved from `qml.qchem` to `qml.ops` because of the generality of the module.